package.json

```diff
@@ -75,7 +75,7 @@
     "eslint": "^7.10.0",
     "eslint-config-prettier": "^6.12.0",
     "jest": "^26.5.2",
-    "jwks-rsa": "^2.0.0",
+    "jwks-rsa": "^2.0.1",
```

Comment:
This change does nothing by the way. I would rather suggest
```
"jwks-rsa": "^2",
```
which is equivalent and would be clearest.